packages/react-auth/package.json

```diff
@@ -2,20 +2,24 @@
   "name": "@pangeacyber/react-auth",
   "description": "Pangea auth provider React component",
   "version": "0.0.8",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
+  "source": "src/index.ts",
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
   "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
   "repository": "https://github.com/pangeacyber/pangea-javascript.git",
   "author": "Pangea (https://pangea.cloud)",
   "license": "MIT",
   "scripts": {
-    "build": "rm -rf dist && rollup -c",
-    "lint": "npx prettier --check .",
-    "prettier:fix": "npx prettier . -w"
+    "build": "parcel build",
```

Comment:
Is the idea to switch every package from rollup to parcel? Just curious what was the main reason? Parcel being zero config is just easier and less error prone?

Reply:
I have the same question, what's the reason for the change?

Reply:
Yeah I went with this in part to try Parcel but more importantly to simplify the current setup, which had been stuck on Rollup v3 for 2 years. I'm hoping with this there will be less packages to keep updated and less config to manage.

Brings this to the other packages would probably be nice but I don't have my sights set on that just yet.